src/pages/catalog.js

```diff
@@ -78,50 +66,104 @@ const FeedstockRowAccordionItem = ({ feedstockId, feedstockSpec }) => {
-                  </Box>
-
-                  {isExpanded ? (
-                    <MinusIcon fontSize='xl' />
-                  ) : (
-                    <AddIcon fontSize='xl' />
-                  )}
-                </AccordionButton>
-
-                <AccordionPanel>
-                  <Text opacity={0.8}>{meta?.description}</Text>
-
-                  <Skeleton isLoaded={!datasetsAreLoading}>
-                    <List spacing={3} my={4}>
-                      {datasets?.map((dataset, index) => (
-                        <DatasetListItem
-                          key={index}
-                          dataset={dataset?.dataset_public_url}
-                        ></DatasetListItem>
-                      ))}
-                    </List>
-                  </Skeleton>
-
-                  <Button
-                    as={Link}
-                    my={4}
-                    href={`/dashboard/feedstock/${feedstockId}`}
-                    colorScheme='teal'
-                    variant='outline'
-                  >
-                    More Details
-                  </Button>
-                </AccordionPanel>
-              </Skeleton>
-            </>
-          )}
-        </AccordionItem>
+        <Box>
+          <Flex alignItems={'center'} justifyContent={'space-between'}></Flex>
+          <Box
+            pos='relative'
+            h='200px'
+            bgImage="linear-gradient(rgba(255, 0, 0, 0.127),rgba(255, 0, 0, 0.1)) , url('')"
+          ></Box>
+          <Box>
+            <Accordion allowMultiple>
+              <AccordionItem>
+                {({ isExpanded }) => (
+                  <>
+                    <AccordionButton>
+                      <Box flex='1' textAlign='left'>
+                        <Skeleton isLoaded={!metaIsLoading}>
+                          <Text>{meta?.title}</Text>
+                        </Skeleton>
+                      </Box>
+
+                      {isExpanded ? (
+                        <MinusIcon fontSize='xl' />
+                      ) : (
+                        <AddIcon fontSize='xl' />
+                      )}
+                    </AccordionButton>
+
+                    <AccordionPanel pb={4}>
+                      <Skeleton isLoaded={!metaIsLoading}>
+                        <Text opacity={0.8}>{meta?.description}</Text>
+                      </Skeleton>
+
+                      <TableContainer my={4}>
+                        <Table variant='striped' size='sm'>
+                          <TableCaption></TableCaption>
+                          <Thead>
+                            <Tr>
+                              <Th></Th>
+                              <Th></Th>
+                              <Th></Th>
+                            </Tr>
+                          </Thead>
+                          <Tbody>
+                            <Tr>
+                              <Td>License</Td>
+                              <Td>:</Td>
```

Comment:
nit: is there a particular reason not to just put this in the `Td` with the row label?
<img width="389" alt="CleanShot 2022-10-11 at 11 09 54@2x" src="https://user-images.githubusercontent.com/12436887/195167394-5a6081fe-8727-4143-adff-91ea11855fd7.png">